### PR TITLE
Add layout colemak_ix

### PR DESCRIPTION
**Pull request by starryturtle**
### Description

layout colemak_ix is added

### CODE REVIEW

**Comment by Miodec** [comment]
Please fix JSON errors: https://github.com/Miodec/monkeytype/runs/5259162893?check_suite_focus=true

**Comment by starryturtle** [comment]
fixed, sorry for not being careful

**Review by starryturtle — COMMENTED** [review]
fixed

File: frontend/static/layouts/_list.json [hunk 1]
```diff
@@ -516,6 +516,17 @@
       "row5": [" "]
     }
   },
+  "colemak_ix": {
+    "keymapShowTopRow": true,
+    "type": "ansi",
+    "keys": {
+      "row1": ["`~", "1!", "2@", "3#", "4$", "5%", "6^", "7&", "8*", "9(", "0)", "=+", "[{"],
+      "row2": ["qQ", "lL", "cC", "mM", "kK", "jJ", "fF", "uU", "yY", ",<;:", "-_", "]}", "\\|"],
```
Comment (Miodec) [inline frontend/static/layouts/_list.json:524]: Row 2 one of the keys exceeds the maximum of 2 characters.

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/layouts/_list.json [hunk 2]
```diff
@@ -1000,4 +1011,4 @@
       "row5": [" "]
   }
```
Comment (Bruception) [inline frontend/static/layouts/_list.json:1012]: This needs to be indented.

**Pull request closed without merging**